sofl/utils/steam_launcher.py

```
@@ -218,11 +231,25 @@ def launch_game(cmd_argv: List[str], env: Dict[str, str], game_dir: Path, in_fla
 
             # Add directory change
             if game_dir:
-                full_cmd = ["sh", "-c", f"cd {shlex.quote(str(game_dir))} && exec \"$@\"", "sh"] + full_cmd[1:]
-
-            logging.info(f"[SOFL] Executing command via flatpak-spawn: {' '.join(shlex.quote(str(arg)) for arg in full_cmd)}")
+                full_cmd = [
+                    "sh",
+                    "-c",
+                    f'cd {shlex.quote(str(game_dir))} && exec "$@"',
+                    "sh",
+                ] + full_cmd[1:]
```

Ruff (0.14.0)

Consider iterable unpacking instead of concatenation

Replace with iterable unpacking

(RUF005)